File: spologp_v/lista2casa-python/lista2casa-python/questao2.py
valor_inteiro = int(input("Digite um valor:"))
numero_chave = 100

resultado = numero_chave-valor_inteiro

if(valor_inteiro>numero_chave):
    print("Não é possível indicar a distância desse valor")
elif(resultado<0):
    print("Esse não é um valor válido, pois a distância entre os valores é negativa")
else:
    print(resultado)

    #O algoritmo solicita ao usuário que digite um valor para indicar a diferença entre ele e o valor chave, no caso, 100. Caso o valor fornecido seja menor que 0, o programa informa que o valor é inválido, pois a distância entre eles é negativa, e caso seja maior que o número chave, também informa que não é possível indicar a diferença entre eles.

File: spologp_v/lista2casa-C/listas 2/lista2sala/listas2casa/lista2casac/questao2.c
#include<stdio.h>

int main()
{
    int valor, valor_chave, distancia;
    printf("Digite um valor:");
    scanf("%i", &valor);
    valor_chave = 100;

    distancia = valor-valor_chave;

if(valor<0){
    distancia = valor*-1-valor_chave;
    printf("A distância desse número para o valor chave é:", valor_chave);
}

else{
    printf("A distância desse número para o valor chave é:", valor_chave);
}

return 0;
} 